config/samples/xgboost-dist/utils.py

```diff
@@ -213,6 +231,37 @@ def dump_model_to_oss(oss_parameters, booster):
         return False
 
     return True
+def dump_model_to_gcp(gcp_parameter,booster):
+    model_fname = os.path.join(tempfile.mkdtemp(), 'model')
+    text_model_fname = os.path.join(tempfile.mkdtemp(), 'model.text')
+    feature_importance = os.path.join(tempfile.mkdtemp(),
+                                      'feature_importance.json')
+
+    gcp_path = gcp_parameters['path']
+    logger.info('---- export model ----')
+    booster.save_model(model_fname)
+    booster.dump_model(text_model_fname)  
+    fscore_dict = booster.get_fscore()
+    with open(feature_importance, 'w') as file:
+        file.write(json.dumps(fscore_dict))
+        logger.info('---- chief dump model successfully!')
```

Comment:
dump model to local ?

Reply:
I learnt it from dump to oss module, I think the logic is dump the model to local first, and then upload from local to the cloud